docs/versioned_docs/version-2.11.0/deprecated/releases-0x.md

```diff
@@ -0,0 +1,291 @@
+:::
+
+**Change summary:**
+
+- Enhanced Storage Policy Enforcement Framework for Jiva.
+- Extend OpenEBS API Server to expose volume snapshot API.
+- Support for deploying OpenEBS via helm charts.
+- Sample Prometheus configuration for collecting OpenEBS Volume Metrics.
+- Sample Grafana OpenEBS Volume Dashboard - using the prometheus Metrics
+
+**Additional details:**
+- [Release Blog](https://openebs.io/blog/openebs-0-5-enables-storage-policies-for-kubernetes-persistent-volumes/)
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/v0.5.0)
+
+## 0.4.0 - Sep 08 2017
+
+:::note Deprecation notice
+This release has been deprecated. Please upgrade to the latest release. See [upgrade instructions](/docs/next/upgrade.html).
+:::
+
+**Change summary:**
+
+- Enhanced MAYA cli support for managing snapshots,usage statistics.
+- Support OpenEBS Maya API Server uses the Kubernetes scheduler logic to place OpenEBS Volume Replicas on different nodes
+- Support Extended deployment of OpenEBS in AWS.
+- Support OpenEBS can be deployed in a minikube setup.
+- Enhanced openebs-k8s-provisioner to recover from crashloopbackoff state
+
+**Additional details:**
+- [Release Blog](https://openebs.io/blog/quick-update-on-openebs-v0-4-a-developer-friendly-release/)
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/v0.4.0)
+
+## 0.3.0 - Jun 29 2017
+
+:::note Deprecation notice
+This release has been deprecated. Please upgrade to the latest release. See [upgrade instructions](/docs/next/upgrade.html).
+:::
+
+**Change summary:**
+
+- Support OpenEBS hyper-converged with Kubernetes Minion Nodes.
+- Enable OpenEBS via the openebs-operator.yaml
+- Supports creation of OpenEBS volumes using Dynamic Provisioner.
+- Storage functionality and Orchestration/Management functionality is delivered as container images on DockerHub.
+
+**Additional details:**
+- [Release Blog](https://openebs.io/blog/openebs-on-the-growth-path-releases-0-3/)
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/v0.3)
+
+## 0.2.0 - Apr 07 2017
+
+:::note Deprecation notice
+This release has been deprecated. Please upgrade to the latest release. See [upgrade instructions](/docs/next/upgrade.html).
+:::
+
+**Change summary:**
+- Integrated OpenEBS FlexVolume Driver and Dynamically Provision OpenEBS Volumes into Kubernetes.
+- Support Maya api server to provides new AWS EBS-like API for provisioning Block Storage.
+- Enhanced Maya api server to Hyper Converged with Nomad Scheduler.
+- Backup/Restore Data from Amazon S3.
+- Node Failure Resiliency Fixes
+
+**Additional details:** 
+- [Release Blog](https://openebs.io/blog/openebs-sprinting-ahead-0-2-released/)
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/v0.2)
+
+
+## See Also:
+
+[OpenEBS Upgrade](/docs/next/upgrade.html) [OpenEBS Releases](/docs/next/releases.html) [OpenEBS FAQ](/docs/next/faq.html) [Container Attached Storage or CAS](/docs/next/cas.html)
```

Comment:
can we fix the links as well as in this PR, all the other navitems are available so it will be easy